netci.groovy

```diff
@@ -85,52 +86,6 @@ pushd %WORKSPACE%\\roslyn-internal
 git submodule init
 git submodule sync
 git submodule update --init --recursive
-init.cmd
-""")
-
-            // Build the SDK and install .NET Core Templates.
-            batchFile("""
-echo *** Build the SDK and install .NET Core Templates  ***
-SET VS150COMNTOOLS=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\Common7\\Tools\\
-SET DeveloperCommandPrompt=%VS150COMNTOOLS%\\VsMSBuildCmd.bat
-
-echo  *** Call VsMSBuildCmd.bat
-call "%DeveloperCommandPrompt%" || goto :BuildFailed "VsMSBuildCmd.bat"
-
-SET VSSDK150Install=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\v15.0\\VSSDK\\
-SET VSSDKInstall=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\v15.0\\VSSDK\\
-
-pushd %WORKSPACE%\\sdk
-echo *** Build SDK
-call build.cmd -Configuration release -SkipTests || goto :BuildFailed "SDK"
-
-SET VSIXExpInstallerExe=%USERPROFILE%\\.nuget\\packages\\roslyntools.microsoft.vsixexpinstaller\\0.2.4-beta\\tools\\VsixExpInstaller.exe
-
-SET VSIXTarget=%WORKSPACE%\\sdk\\bin\\Release\\Microsoft.VisualStudio.ProjectSystem.CSharp.NetStandard.Templates.vsix
-echo *** Install %VSIXTarget%
-%VSIXExpInstallerExe% /rootsuffix:RoslynDev %VSIXTarget%
-if not "%ERRORLEVEL%"=="0" echo ERROR: %VSIXTarget% did not install successfully
-
-SET VSIXTarget=%WORKSPACE%\\sdk\\bin\\Release\\Microsoft.VisualStudio.ProjectSystem.CSharp.Templates.vsix
-echo *** Install %VSIXTarget%
-%VSIXExpInstallerExe% /rootsuffix:RoslynDev %VSIXTarget%
-if not "%ERRORLEVEL%"=="0" echo ERROR: %VSIXTarget% did not install successfully
-
-SET VSIXTarget=%WORKSPACE%\\sdk\\bin\\Release\\Microsoft.VisualStudio.ProjectSystem.VisualBasic.NetStandard.Templates.vsix
-echo *** Install %VSIXTarget%
-%VSIXExpInstallerExe% /rootsuffix:RoslynDev %VSIXTarget%
-if not "%ERRORLEVEL%"=="0" echo ERROR: %VSIXTarget% did not install successfully
-
-SET VSIXTarget=%WORKSPACE%\\sdk\\bin\\Release\\Microsoft.VisualStudio.ProjectSystem.VisualBasic.Templates.vsix
-echo *** Install %VSIXTarget%
-%VSIXExpInstallerExe% /rootsuffix:RoslynDev %VSIXTarget%
-if not "%ERRORLEVEL%"=="0" echo ERROR: %VSIXTarget% did not install successfully
-
-exit /b 0
-
-:BuildFailed
-echo %1 - Build failed with ERRORLEVEL %ERRORLEVEL%
-exit /b 1
 """)
```

Comment:
Please don't merge this in - we want to use the latest SDK bits for testing project system.